### PR TITLE
fixes #2607 - add autocomplete=false attribute to root_host_pass input

### PR DESCRIPTION
**Pull request by lzoubek**

### CODE REVIEW

File: app/views/hosts/_operating_system.html.erb [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   <%= textarea_f f, :disk, :class => "input-xxlarge", :rows => "4",
            :help_block => _("What ever text(or ERB template) you use in here, would be used as your OS disk layout options If you want to use the partition table option, delete all of the text from this field") %>
-  <%= password_f f, :root_pass %>
+  <%= password_f f, :root_pass, :autocomplete => "off" %>
```
Comment (ohadlevy) [inline app/views/hosts/_operating_system.html.erb]: thanks @lzoubek!

I think the better way would be to actually change that to all password fields inside the password_f method?

Comment (isratrade) [inline app/views/hosts/_operating_system.html.erb]: @lzoubek, you can find password_f in app/helpers/layout_helper.rb

**Comment by lzoubek** [comment]
now it should affect all password fields, "usual" login field is not affected

File: app/helpers/layout_helper.rb [hunk 2]
```diff
@@ -44,6 +44,7 @@ def textarea_f(f, attr, options = {})
 
   def password_f(f, attr, options = {})
     field(f, attr, options) do
+      options[:autocomplete] = "off"
```
Comment (ohadlevy) [inline app/helpers/layout_helper.rb]: looks good, one minor issue, I would like to be able to override it, so maybe you can add the option only if it doesnt exist?  e.g.

``` ruby
 options[:autocomplete] ||= "off"
```

**Comment by domcleal** [comment]
Thanks for the patch @lzoubek, merged as 0c6065e for Foreman 1.2.

**Comment by lzap** [comment]
@lzoubek you made my day yesterday. We was struggling with this bug for a week or something. :-D Thanks!

**Comment by isratrade** [comment]
+1

**Pull request closed without merging**